pkgs/games/dwarf-fortress/wrapper/default.nix

```diff
@@ -101,5 +80,26 @@ stdenv.mkDerivation rec {
     chmod 755 $out/bin/soundsense
   '';
 
+  postBuild = ''
+    # De-symlink init.txt
+    cp $out/data/init/init.txt init.txt
+    rm $out/data/init/init.txt
+    mv init.txt $out/data/init/init.txt
+  '' + lib.optionalString enableDFHack ''
+    rm $out/hack/symbols.xml
+    echo "[$out/hack/symbols.xml] $(cat ${dwarf-fortress}/hash.md5.orig) => $(cat ${dwarf-fortress}/hash.md5)"
```

Comment:
Can't figure out why this is never echoing. It seems like this started happening fairly recently, too - dfhack used to work just fine!